docs/docs/geopyspark.geotrellis.key_conversion.rst

```diff
@@ -0,0 +1,6 @@
+geopyspark.geotrellis.key_conversion module
```

Comment:
Could have this listed in `geopyspark.geotrellis.rst` under the `Modules` tree (it's at the bottom of the file). That way that document will appear in docs directory on the left side of the page.